qiskit_experiments/library/quantum_volume/qv_analysis.py

```diff
@@ -37,26 +37,14 @@ class QuantumVolumeAnalysis(BaseAnalysis):
         is the success probability.
     """
 
-    # pylint: disable = arguments-differ
-    def _run_analysis(
-        self,
-        experiment_data,
-        plot: bool = True,
-        ax: Optional["matplotlib.pyplot.AxesSubplot"] = None,
-    ):
-        """Run analysis on circuit data.
-
-        Args:
-            experiment_data (ExperimentData): the experiment data to analyze.
-            plot (bool): If True generate a plot of fitted data.
-            ax (AxesSubplot): Optional, matplotlib axis to add plot to.
+    @classmethod
+    def _default_options(cls) -> Options:
+        return Options(
+            plot=True,
+            ax=None,
+        )
```

Comment:
```suggestion
    def _default_options(cls) -> Options:
        """Return default analysis options.

        Analysis Options:
            plot (bool): Set ``True`` to create figure for fit result.
            ax(AxesSubplot): Optional. A matplotlib axis object to draw. 
        """
        options = super()._default_options()
        options.plot = True
        options.ax = None
        
        return options
```
for autodoc

Reply:
These docs won't be shown in any API docs since its an internal method. Also super has no options to inherit so I think it's clearer to not call the super method.

Reply:
Yes, but the autodoc will parse this method documentation to generate analysis option description.
https://github.com/Qiskit/qiskit-experiments/blob/2a8aad017c61562b3d89857e69ddb1ff36471bed/docs/_ext/custom_styles/styles.py#L319-L324
This is good place to recursively parse argument docs from superclass.

Reply:
Oh cool, didn't realise this. Is this documented somewhere?